core/mount/mount_windows.go

```diff
@@ -130,6 +141,40 @@ func (m *Mount) GetParentPaths() ([]string, error) {
 	return parentLayerPaths, nil
 }
 
+// gets the paths of the parent cims of this mount
+func (m *Mount) GetParentCimPaths() ([]string, error) {
```

Comment:
I would prefer these functions be defined in the cimfs snapshotter package, and take a `Mount` as input, rather than being attached to the type. We probably don't want `Mount` to end up with functions hanging off of it for specific mount types, it should stay generic.